packages/dx-react-scheduler-material-ui/src/templates/utils.js

```diff
@@ -42,3 +42,5 @@ export const getAppointmentColor = (level, color, defaultColor) => {
   if (typeof color === 'string') return color;
   return ensureColor(level, color);
 };
+
+export const addCommaAndSpaceToString = string => !!string && `${string},\xa0`;
```

Comment:
Should we return a non-bulean value? 
![image](https://user-images.githubusercontent.com/15093892/70709569-38440f00-1cee-11ea-854d-0b52d3722fd0.png)

```js
export const addCommaAndSpaceToString = string => string ? `${string},\xa0` : null;
```